osmtm/templates/start.mako

```diff
@@ -10,17 +10,17 @@ login_url= request.route_path('login', _query=[('came_from', request.url)])
 <div class="container">
   <div class="row">
     <h2>
-      Welcome, new user!
+      ${_('Welcome, new user!')}
     </h2>
     <div>
       <p>
-        You're ready to use the Tasking Manager.
+        ${_('You\'re ready to use the Tasking Manager.')}
       </p>
       <p>
-        Please <a href="${login_url}">login</a>.
+        ${_('Please')} <a href="${login_url}">${_('login')}</a>
```

Comment:
I agree, but it failed when I had a single sentence. Feel free to correct.

2015-06-09 13:35 GMT+02:00 Pierre GIRAUD <notifications@github.com>:

> In osmtm/templates/start.mako
> <https://github.com/hotosm/osm-tasking-manager2/pull/641#discussion_r32003827>
> :
>
> >        </p>
> >        <p>
> > -        Please <a href="${login_url}">login</a>.
> > +        ${_('Please')} <a href="${login_url}">${_('login')}</a>
>
> It would be better if this sentence is not splitted into two.
> "please login" is probably translated differently than "please" and
> "login".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hotosm/osm-tasking-manager2/pull/641/files#r32003827>.
>



-- 
 [image: http://www.dogodigi.net] <http://www.dogodigi.net>
*Milo van der Linden*
web: dogodigi <http://www.dogodigi.net>
tel: +31-6-16598808